Cartfile.resolved

```diff
@@ -7,6 +7,6 @@ github "RxSwiftCommunity/RxDataSources" "3.0.2"
 github "RxSwiftCommunity/RxOptional" "3.4.1"
 github "SwiftyJSON/SwiftyJSON" "3.1.4"
 github "jrendel/SwiftKeychainWrapper" "3.0.1"
-github "mozilla-lockbox/lockbox-ios-fxa-sync" "5a51446813ef29d8adb9e3f3755ee64db9235fc5"
+github "mozilla-lockbox/lockbox-ios-fxa-sync" "e5b19f0a28504f71dd9ac515446c92647cac1dc9"
```

Comment:
For easy reference, a link: https://github.com/mozilla-lockbox/lockbox-ios-fxa-sync/commit/e5b19f0a28504f71dd9ac515446c92647cac1dc9